# -*- coding: utf-8 -*-
# Generated by Django 1.11 on 2018-04-16 08:10
from __future__ import unicode_literals

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('data', '0006_auto_20180319_1010'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='datarepository',
            name='path',
        ),
        migrations.AddField(
            model_name='datarepository',
            name='dns',
            field=models.CharField(blank=True, help_text='DNS of the network drive', max_length=200, validators=[django.core.validators.RegexValidator('^[a-zA-Z0-9\\.\\-\\_]+$', code='invalid_dns', message='Invalid DNS')]),
        ),
        migrations.AddField(
            model_name='datarepository',
            name='globus_path',
            field=models.CharField(blank=True, help_text='absolute path to the repository on the server e.g. /mnt/something/', max_length=1000),
        ),
        migrations.AddField(
            model_name='datarepository',
            name='timezone',
            field=models.CharField(blank=True, default='GB', help_text='Timezone of the server (see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)', max_length=64),
        ),
        migrations.AlterField(
            model_name='filerecord',
            name='relative_path',
            field=models.CharField(blank=True, help_text='path name within repository', max_length=1000, validators=[django.core.validators.RegexValidator('^[a-zA-Z0-9\\_][^\\\\\\:]+$', code='invalid_path', message='Invalid path')]),
        ),
    ]
